google/cloud/compute/doc/main.dox

```diff
@@ -24,16 +24,106 @@ which should give you a taste of the Cloud Compute Engine API C++ client library
 
 <!-- inject-client-list-start -->
 This library offers multiple `*Client` classes, which are listed below. Each
-one of these classes exposes all the RPCs for a REST `resource` as member
-functions of the class. This library groups multiple REST services because they
+one of these classes exposes all the RPCs for a gRPC `service` as member
+functions of the class. This library groups multiple gRPC services because they
 are part of the same product or are often used together. A typical example may
```

Comment:
Hm, maybe we don't want this change.

Reply:
I propose that we merge this PR, then in a follow up, change the checkers script to `s/gRPC //` in these two places.